Modules/_testcapimodule.c

```diff
@@ -3951,6 +3951,10 @@ PyInit__testcapi(void)
     PyModule_AddIntConstant(m, "the_number_three", 3);
     PyModule_AddIntMacro(m, Py_C_RECURSION_LIMIT);
 
+    PyModule_AddIntConstant(m, "Py_single_input", Py_single_input);
+    PyModule_AddIntConstant(m, "Py_file_input", Py_file_input);
+    PyModule_AddIntConstant(m, "Py_eval_input", Py_eval_input);
```

Comment:
Add error handling[^1]:

```suggestion
    if (PyModule_AddIntConstant(m, "Py_single_input", Py_single_input) < 0) {
        return NULL;
    }
    if (PyModule_AddIntConstant(m, "Py_file_input", Py_file_input) < 0) {
        return NULL;
    }
    if (PyModule_AddIntConstant(m, "Py_eval_input", Py_eval_input) < 0) {
        return NULL;
    }
```

[^1]: I know the rest of this setup function has sloppy error handling; we should still ensure proper error handling when adding new code. A PR for fixing up the existing sloppy error handling is welcome :)